<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createPlotFromData</class>
 <widget class="QDialog" name="createPlotFromData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Plot From Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="2">
    <widget class="QPushButton" name="cmdCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="cmdCreate">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Raw Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="txtDataToPlot">
        <property name="maxLength">
         <number>100000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Delimiter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtDelimiter"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chkAutoDelim">
            <property name="text">
             <string>Auto Delimiter</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLineEndAsDelim">
            <property name="text">
             <string>Line Endings As Dimiter</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Base</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbBase">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Decimal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hex</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblHexType">
            <property name="text">
             <string>Hex Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbHexType">
            <item>
             <property name="text">
              <string>Unsigned Int</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Signed 8 Bit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Signed 16 Bit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Signed 32 Bit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Signed 64 Bit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float 32 Bit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float 64 Bit</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Num Points</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtNumPoints">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>New Plot / Curve Name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbPlotName">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plot Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Curve Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtCurveName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
